api/HotelsDatabaseContext.cs

```diff
@@ -36,6 +37,29 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                     .IsUnicode(false)
                     .IsFixedLength(true);
             });
+
+            modelBuilder.Entity<Hotel>(entity =>
+            {
+                entity.Property(e => e.Address)
+                    .IsRequired()
+                    .HasMaxLength(155)
+                    .IsUnicode(false);
+
+                entity.Property(e => e.Country)
+                    .IsRequired()
+                    .HasMaxLength(60)
+                    .IsUnicode(false);
+
+                entity.Property(e => e.Name)
+                    .IsRequired()
+                    .HasMaxLength(60)
+                    .IsUnicode(false);
+
+                entity.Property(e => e.Sity)
+                    .IsRequired()
+                    .HasMaxLength(60)
```

Comment:
Why `60`? Is it the longest name of the city on Earth? :)

Reply:
The longest name I have found was 52 characters, but now I googled one more time and it should have 85 characters max